features/promotion/managing_promotions/adding_promotion.feature

```diff
@@ -62,7 +62,7 @@ Feature: Adding a new promotion
         Given I want to create a new promotion
         When I specify its code as "FULL_METAL_PROMOTION"
         And I name it "Full metal promotion"
-        And I specify that it should starts at "21.04.2017"
-        And I specify that it should ends at "21.05.2017"
+        And I specify that it should start at "21.04.2017"
+        And I specify that it should end at "21.05.2017"
```

Comment:
Promotion is not really available, we could say:

I make it active
I set its start date to

Something like that.